src/components/View/Struttura/StrutturaPersonale.jsx

```diff
@@ -3,59 +3,40 @@ import { defineMessages, useIntl } from 'react-intl';
 import { RichTextSection } from 'io-sanita-theme/helpers';
 import { CardPersona } from 'io-sanita-theme/components';
 import { Row, Col } from 'design-react-kit';
+import { BackReferences } from 'io-sanita-theme/components/View/commons';
 
 const messages = defineMessages({
   personale: {
     id: 'struttura_personale',
-    defaultMessage: 'Personale',
-  },
-  coordinatore: {
-    id: 'struttura_coordinatore',
-    defaultMessage: 'Coordinatore',
+    defaultMessage: 'Personale della struttura',
   },
 });
 
-const UOPersonale = ({ content }) => {
+const StrutturaPersonale = ({ content }) => {
   const intl = useIntl();
 
-  return (content?.personale_correlato?.length > 0 ||
```

Comment:
il campo 'personale_correlato' non esiste piu? 
Se prima esisteva e adesso è cambiato il nome, è stata pensato un script di upgrade sul BE?